pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation_test.go

```diff
@@ -1080,29 +1080,30 @@ func TestInjectLibConfig(t *testing.T) {
 
 func TestInjectLibInitContainer(t *testing.T) {
 	tests := []struct {
-		name    string
-		cpu     string
-		mem     string
-		pod     *corev1.Pod
-		image   string
-		lang    language
-		wantErr bool
-		wantCPU string
-		wantMem string
-		secCtx  *corev1.SecurityContext
+		name              string
+		cpu               string
+		mem               string
+		pod               *corev1.Pod
+		image             string
+		lang              language
+		wantSkipInjection bool
+		wantErr           bool
+		wantCPU           string
+		wantMem           string
+		secCtx            *corev1.SecurityContext
 	}{
 		{
-			name:    "no resources, no security context",
+			name:    "no_resources,no_security_context",
 			pod:     common.FakePod("java-pod"),
 			image:   "gcr.io/datadoghq/dd-lib-java-init:v1",
 			lang:    java,
 			wantErr: false,
-			wantCPU: "50m",
-			wantMem: "100Mi",
+			wantCPU: "0",
+			wantMem: "0",
```

Comment:
Should we keep setting the min value even if no resources are set on other containers? 

I'm a bit worry about some edge cases where a user have another mutation admission controller that set the request/limit for a defined set of containers. Then they have a validation container that enforce request/limit configuration on all containers. 
I don't have a real customers setup in mind, but it could happen with a low probability.